job/server/src/main/java/alluxio/worker/job/command/JobWorkerHealthReporter.java

```diff
@@ -34,7 +41,17 @@ public JobWorkerHealthReporter() {
    *
    * @return the system load average of the worker
    */
-  public double[] getCpuLoadAverage() {
-    return mProcessor.getSystemLoadAverage(3);
+  public List<Double> getCpuLoadAverage() {
+    return DoubleStream.of(mHardware.getProcessor().getSystemLoadAverage(3)).boxed()
+        .collect(Collectors.toList());
+  }
+
+  /**
+   * @return Network health for each of the network interfaces of the worker
+   */
+  public List<NetworkHealth> getNetworkHealths() {
+    return Arrays.stream(mHardware.getNetworkIFs()).map((networkIF) ->
+        new NetworkHealth(networkIF.getName(), networkIF.getBytesRecv(), networkIF.getBytesSent(),
+            networkIF.getSpeed())).collect(Collectors.toList());
```

Comment:
Is this network speed the max connection speed or average?

My main concern is that I'm not sure if having this information is useful if we can't tell which network interface our traffic is traveling through.

Reply:
My current thought is that I'm going to use the network interface with the highest speed/bytesRecv, etc.

Reply:
How do you plan on "choosing" the network interface?

Reply:
The way I'm considering this is that I'm not necessarily going to choose a network interface but rather just look at the fact that overall traffic is high or in a different case, look at the difference when there are no active tasks running.

Reply:
how do you plan on tracking all of this information?

Reply:
After discussing with Calvin, both me and him agree that in practice we only expect to see one network interface to be used. (By that, we mean that everything except for one will have very little bytes recv/sent). So it will be very easy to later filter out for actual logic.

Reply:
I guess my main concern is that the metrics that this PR adds just don't seem useful in most situations.

Currently, it looks like the worker collects the bare minimum (if speed, bytes sent/recv) and then sends them to the master. But the master can't really do anything except relay the information to a user. And even then, it is unlikely that the information about the interface is really of any use to the user.

I guess this PR does at least set up the scaffolding for adding better metrics in the future. One thing that I think would be useful is to have 1/5/15 minute averages for bytes sent/sec recv/sec.

That can be left for another PR though